tests/test_cube_equaliser.py

```diff
@@ -1,133 +1,192 @@
-    assert "creation_date" not in test_attr
-    assert "history" not in test_attr
-    assert "tracking_id" not in test_attr
-
+import cube_helper as ch
+
+
+class TestCubeEqualiser(unittest.TestCase):
+
+    def setUp(self):
+        super(TestCubeEqualiser, self).setUp()
+        abs_path = os.path.dirname(os.path.abspath(__file__))
+        self.tmp_dir = abs_path + '/' + 'tmp_dir/'
+        self.tmp_dir_aux = abs_path + '/' + 'tmp_dir_aux/'
+        self.tmp_dir_attr = abs_path + '/' + 'tmp_dir_attr/'
+        if not os.path.exists(self.tmp_dir):
+            os.mkdir(self.tmp_dir)
+        if not os.path.exists(self.tmp_dir_attr):
+            os.mkdir(self.tmp_dir_attr)
+        if not os.path.exists(self.tmp_dir_aux):
+            os.mkdir(self.tmp_dir_aux)
+        base_cube = stock.realistic_3d()
+        cube_1 = base_cube[0:2]
+        cube_2 = base_cube[2:4]
+        cube_3 = base_cube[4:]
+        self.temp_1 = 'temp_1.nc'
+        self.temp_2 = 'temp_2.nc'
+        self.temp_3 = 'temp_3.nc'
+        iris.save(cube_1, self.tmp_dir + self.temp_1)
+        iris.save(cube_2, self.tmp_dir + self.temp_2)
+        iris.save(cube_3, self.tmp_dir + self.temp_3)
+        cube_1 = base_cube[0:2]
+        cube_2 = base_cube[2:4]
+        cube_3 = base_cube[4:]
+        cube_2.attributes['history'] = 'some differing attributes'
+        cube_2.attributes['creation_date'] = 'life day'
+        cube_3.attributes['tracking_id'] = 'askbjQODOQwd210934AS'
+        self.temp_1_attr = 'temp_1_attr.nc'
+        self.temp_2_attr = 'temp_2_attr.nc'
+        self.temp_3_attr = 'temp_3_attr.nc'
+        iris.save(cube_1, self.tmp_dir_attr + self.temp_1_attr)
+        iris.save(cube_2, self.tmp_dir_attr + self.temp_2_attr)
+        iris.save(cube_3, self.tmp_dir_attr + self.temp_3_attr)
+        cube_1 = base_cube[0:2]
+        cube_2 = base_cube[2:4]
+        cube_3 = base_cube[4:]
+        height_coord = iris.coords.AuxCoord(2,
+                                            standard_name='height',
+                                            long_name='height',
+                                            var_name='height',
+                                            units='m')
+        cube_2.add_aux_coord(height_coord)
+        self.temp_1_aux = 'temp_1_aux.nc'
+        self.temp_2_aux = 'temp_2_aux.nc'
+        self.temp_3_aux = 'temp_3_aux.nc'
+        iris.save(cube_1, self.tmp_dir_aux + self.temp_1_aux)
+        iris.save(cube_2, self.tmp_dir_aux + self.temp_2_aux)
+        iris.save(cube_3, self.tmp_dir_aux + self.temp_3_aux)
+
+    def test_equalise_attributes(self):
+        glob_path = self.tmp_dir_attr + '*.nc'
+        print(glob_path)
+        filepaths = glob(glob_path)
+        test_load = [iris.load_cube(cube) for cube in filepaths]
+        test_load = ch.equalise_attributes(test_load)
+        for cubes in test_load:
+            self.assertEqual(cubes.attributes, test_load[0].attributes)
+
+    def test_equalise_time_units(self):
```

Comment:
I think that in this test, because all of the test cubes have been generated from time slices of the same stock cube, they all have the same time units to start with. In `settUp()`, or at the start of this function, can you modify the time units of one cube to check that `ch.equalise_attributes()` is equalising the time units please?